CONTRIBUTING.md

```diff
@@ -9,6 +9,16 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
 
 Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
 
+Want to chat with Probot users and contributors? [Join us in Slack](https://probot-slackin.herokuapp.com/)!
```

Comment:
Do you think it's best to link to our slack here, or to the https://probot.github.io/community section of the website which includes the slack link?

Reply:
I think you're right; the community page is better! Updated.

Reply:
You updated the other instance but not this one. Should be: 
> `Want to get more involved with the Probot community? [Take a look at our community page](https://probot.github.io/community/)!`